### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by quintrino** [deivid-rodriguez/pry-byebug#375]
Hi, 

Looks like there's been at least 30+ commits merged since the last commit. Any chance of a release so that it's updated on rubygems?

**Comment by milosivanovic** [comment]
Yeah, I'd like this too.

I'm seeing:

```
/root/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/pry-byebug-3.8.0/lib/pry-byebug/helpers/navigation.rb:13: warning: _pry_ is deprecated, use pry_instance instead
/root/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/pry-byebug-3.8.0/lib/pry-byebug/helpers/navigation.rb:15: warning: _pry_ is deprecated, use pry_instance instead
```

but that was fixed a year ago in https://github.com/deivid-rodriguez/pry-byebug/commit/b16dea0cf934760797270288fb995b6bf32dbf69 which still hasn't been released.

**Comment by deivid-rodriguez** [comment]
I released 3.10.0, let me know if it works!
